docs/guide/app.md

```diff
@@ -147,7 +148,39 @@ The addition of `[str]` tells mypy that `run()` is expected to return a string.
 
     Type annotations are entirely optional (but recommended) with Textual.
 
-## CSS
+## Customising your app
+
+### Setting the title and subtitle
+
+By default, the title of your application matches the name of the application class and the subtitle is empty.
```

Comment:
It's perhaps not clear to the reader yet what "title" and "subtitle" are. Suggest an introduction something along these line:

"Textual apps have a "title" attribute which is the name of your app, and a "subtitle" attribute which is additional title text that indicates the context of your app. Both values are will be displayed in the apps Header (if there is one)."